pkg/controllers/updaterun/execution_integration_test.go

```diff
@@ -785,47 +785,71 @@ var _ = Describe("UpdateRun execution tests - double stages", func() {
 			validateUpdateRunMetricsEmitted(wantMetrics...)
 		})
 
-		It("Should stop the in middle of cluster updating when update run state is Stop", func() {
+		It("Should be stopping the in middle of cluster updating when update run state is Stop", func() {
```

Comment:
Please move stop related tests to a new file `stop_integration_test.go`, thanks!